### ISSUE TITLE
Сan't send funds from added account

### ISSUE THREAD

**Issue by asemiankevich** [status-im/status-react#8759]
# Problem

Cant send funds from added account - `"{\"error\":{\"code\":4,\"message\":\"transaction can only be send by its creator\"}}"`

![image](https://user-images.githubusercontent.com/28752642/63082688-9256e280-bf4f-11e9-9260-fe9795a513de.png)

## Steps:

- create new account in status
- switch to ropsten and faucet some funds
- open wallet and add new account
- send some funds to the new account
- open this new account and send some funds back to your status account 

@flexsurfer @gravityblast

**Comment by gravityblast** [comment]
@asemiankevich @flexsurfer can we have an example of the exact data we send to `SendTransaction` to understand it better?

**Comment by flexsurfer** [comment]
@gravityblast nothing special, the only thing is `from` is not default wallet
`Statusgo.sendTransaction(txArgsJSON, password);`
```
txArgsJSON : {"to":"0xc55cf4b03948d7ebc8b9e8bad92643703811d162","from":"0x4eEC532DbD191C2186A2b50fFA63De79ea51aF33","data":"0xa9059cbb000000000000000000000000c0A55eB8233d99B653Cb68296474277FCcD6780D000000000000000000000000000000000000000000000002b5e3af16b1880000","gas":"0x1a9d3","gasPrice":"0x3b9aca00"}
```
if `from` is default wallet it works fine

**Comment by gravityblast** [comment]
I know why know. When you login, you pass `mainAccountAddress`, `watchAddresses`. 
If the address is you are using is not among those, it doesn't work.
I guess we want to add a binding to be able to add new addresses to the `watchAddresses` list right? do you already add them somehow with the wallet service? otherwise you don't even get updates on their balance right? @flexsurfer

**Comment by flexsurfer** [comment]
yeah, so maybe better to look at this again after https://github.com/status-im/status-go/issues/1566

**Comment by yenda** [comment]
@asemiankevich can you reproduce in develop even if you logout/login after adding the new account? I fixed that in #8832 but it still need a fix in status-go for not having to relogin
